doc/elements-introduction.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Introduction
+order: 1
+layout: page
+---
+
+# Introduction
+
+## Vaadin Core Elements
+
+Vaadin Core Elements is a free and open source set of high quality Web Components for building mobile and desktop web applications in modern browsers. It builds on top of Google’s Polymer library, and augments the Iron, Paper and other element sets of Polymer with elements that are needed in building business applications.
+
+Although based on Polymer, Vaadin elements can be used together with any other web framework which has support for Web Components, including Angular 2, React, Ember, Meteor and GWT.
+
+Current set of Vaadin Core Elements include:
+
+ - [vaadin-grid](https://github.com/vaadin/vaadin-grid)
+ - [vaadin-combo-box](https://github.com/vaadin/vaadin-combo-box)
+
+Learn more about the benefits of using Vaadin elements from the [Vaadin elements page »](https://vaadin.com/elements)
+
+### Web Components
```

Comment:
I would nest these two section in hight level one like:
- Vaadin Core Elements
- Background
  - Web Components
  - Polymer
- Browser Compatilibity (this could be under background or at this level)